internal/local/lps.go

```diff
@@ -57,7 +57,6 @@ func ExecuteCommand(flags *flags.Flags) int {
 	return resultCode
 }
 
-func (service *ProvisioningService) setupWsmanClient(username string, password string) bool {
+func (service *ProvisioningService) setupWsmanClient(username string, password string) {
 	service.client = wsman.NewClient(service.serverURL, username, password, true)
```

Comment:
I'm not understanding the removing of the return bool.  I know we are always returning true, so probably not helpful, unless the calling function needs to know that the wsman Client is setup and ready.